lib/hipchat/user.rb

```diff
@@ -100,5 +99,36 @@ def delete(params = {})
       true
     end
 
+    #
+    # Update status.
+    # API: https://www.hipchat.com/docs/apiv2/method/update_user
+    # Request body
+    # name - User's full name.  Valid length range: 1-50
+    # status - string may be null
+    # show - string - the status to show for the user. Available options 'away', 'chat', 'dnd', 'xa'
+    # mention_name - User's @mention name without the @
+    # email - User's email
+    def status(message, options = {})
+      name  = options[:name]    ? options[:name] : 'test user'
+      status        = options[:status]  ? options[:status] : 'Away'
+      show          = options[:show]    ? options[:show] : 'dnd' 
+      mention_name  = options[:mention_name] ? options[:mention_name] : 'testuser'
+      email         = options[:email] ? options[:email] : 'testuser'
+
+
+      response = self.class.put(@api.view_config[:url],
+                                 :query => { :auth_token => @token },
+                                 :body => {
```

Comment:
Will not including all fields cause the user's title, timezone and other missing fields to be wiped out?  (Password should be OK.)